core/src/main/java/com/datastax/dse/driver/api/core/config/DseDriverOption.java

```diff
@@ -288,6 +288,13 @@ public enum DseDriverOption implements DriverOption {
    * <p>Value-type: {@link java.time.Duration Duration}
    */
   METRICS_NODE_GRAPH_MESSAGES_SLO("advanced.metrics.node.graph-messages.slo"),
+
+  /**
+   * The batch type that is set in the configuration file
+   *
+   * <p>Value-type: {@link String}
+   */
+  BATCH_TYPE_CONFIGURATION("advanced.batch-type-configuration"),
```

Comment:
This enum represents the set of config values for options specific to DSE.  We want this config to be available for Cassandra users as well so it really should be in com.datastax.oss.driver.api.core.config.DefaultDriverOption instead.

Reply:
at first , I just put this enum variable in DefaultDriverOption for I think it should also be used by c* users, But after I read the jira agagin I found out that The description of the jira is "FedEx (David Twynam) would like the ability to set batch-type application-wide ", I think FedEx may be one of datastax's customers ，then I change the  BATCH_TYPE_CONFIGURATION to DseDriverOption , I will change it back to DefaultDriverOption.